tests/unit_tests/scan_tests.sh

```diff
@@ -1,27 +1,30 @@
 #! /bin/bash
 # shellcheck disable=SC2155
 
-source ./entrypoint.sh "" "style" "--test"
+source ./entrypoint.sh "" "style" "" "--test"
 
-# scan_file() tests
+# scan_file tests
 test_scan_valid_script_with_extension(){
     local expected="Scanning sample.bash"
     local actual=$(scan_file ./test_data/script_type/sample.bash)
+
     assertContains "Actual messages:$actual Did not contain the expected message.\n" "$actual" "$expected"
 }
 
 test_scan_valid_script_without_extension(){
     local expected="Scanning executable_script"
     local actual=$(scan_file ./test_data/test_dir/executable_script)
+
     assertContains "Actual messages:$actual Did not contain the expected message.\n" "$actual" "$expected" 
 }
 
 test_scan_unsuppoted_script(){
```

Comment:
Oops, we missed a typo from previous PRs

Reply:
good catch!!